writing/linecount.tex

```diff
@@ -0,0 +1,13 @@
+
+\begin{tabular}{ |s|p{2cm}| }
```

Comment:
I honestly think this probably should be committed. I see no reason as to why it is a missing file and needs to be regenerated on every computer.

It may want to get generated on every push to main, but I don't now if I see a reason to ignore it. I can if you'd like though, and then when we import it from `main.tex` it will fail to compile if you haven't generated the table yet

Reply:
Any files that can be generated should be generated
"push button eval" philosophy that has served me well in the past

Reply:
I'm going to insist we don't commit this file

Reply:
I was very confused on what you meant by "push button eval", I thought that'd mean keep it in.

I'm very happy to get rid of the file, lets keep a .gitignore inside this directory for any generated latex files